api_docs/changelog.md

```diff
@@ -39,7 +48,7 @@ format used by the Zulip server that they are interacting with.
 
 **Feature level 179**
 
-* [`POST /scheduled_messages`](/api/create-or-update-scheduled-message):
+* [`POST /scheduled_messages`](/api/create-scheduled-message):
```

Comment:
We perhaps need a redirect for this?

Reply:
I don't think so, it wasn't around for long and I think there aren't any links yet.